lib/components/enter-a-domain-component.js

```diff
@@ -0,0 +1,21 @@
+import { By } from 'selenium-webdriver';
+
+import BaseContainer from '../base-container.js';
+import * as driverHelper from '../driver-helper';
+
+export default class EnterADomainComponent extends BaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.map-domain-step__add-domain' ) );
+	}
+
+	enterADomain() {
+		const blogName = 'myawesomedomain.com';
+		const entryInputSelector = By.className( 'map-domain-step__external-domain' );
+		driverHelper.setWhenSettable( this.driver, entryInputSelector, blogName );
+	}
+
+	clickonAddButtonToAddDomainToTheCart() {
+		const addDomainSelector = By.css( '.map-domain-step__go.button.is-primary' );
+		return driverHelper.clickWhenClickable( this.driver, addDomainSelector, this.explicitWaitMS );
```

Comment:
It doesn't hurt, but you don't need to explicitly pass `this.explicitWaitMS` here since it has a default value (in earlier code samples it may not have had a default value, so there are examples like this throughout the code still)

Reply:
I was wondering about `this.explicitWaitMS` when writing this function but decided to leave it _as is_ thinking I might miss some cases where it is needed (that I am not aware of) if I remove it. Will ask clarifying questions next time. Fixed.